### PR TITLE
Added PullRequestTemplate.md

### LINKED ISSUE #70: Add Pull Request Template
**Is your feature request related to a problem? Please describe.**
Currently no pull request template is available in this project.

**Describe the solution you'd like**
@krharsh17 I am a GSSOC participant and I would like to add a pull_request_template.md file in this project.

**Additional context**
### Pull request
pull request

### PR DESCRIPTION
**Pull request by Aman-Codes**
Closes: #70 
@krharsh17 Added the pull request template. Please review it and do let me know if any changes sre required.

### CODE REVIEW

File: .github/PullRequestTemplate.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+## Issue that this pull request solves
+
+Closes: # (issue number)
```
Comment (samagragupta) [inline .github/PullRequestTemplate.md]: Replace `Closes` with `issue`

**Comment by Aman-Codes** [comment]
@samagragupta Did the asked changes.

File: .github/PullRequestTemplate.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ## Issue that this pull request solves
 
-Closes: # (issue number)
+Issue: # (issue number)
```
Comment (ShivangiSingh17) [inline .github/PullRequestTemplate.md]: "Fixes" will be the exact word to mention.

**Review by ShivangiSingh17 — DISMISSED** [review]

**Review by ShivangiSingh17 — DISMISSED** [review]

**Review by krharsh17 — COMMENTED** [review]
@Aman-Codes please squash all your commits into one.

**Review by krharsh17 — APPROVED** [review]

**Pull request merged**